### PR TITLE
tag: Automatically create stable branch

### PR DESCRIPTION
**Pull request by jcvenegas**
After a rc0 is created the path to have an stable release starts, after that
any rc0 is to improve stabability and not more features are added. When it is
the projects is considered stable no more rc* are done.

Fixes: #611

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: release/tag_repos.sh [hunk 1]
```diff
@@ -122,6 +124,17 @@ tag_repos() {
 			info "Creating tag ${tag} for ${repo}"
 			git tag -a "${tag}" -s -m "${PROJECT} release ${tag}"
 		fi
+		if [[ ! ${not_stable_branch[*]} =~ ${repo} ]] && [ "${branch}" == "master" ]; then
+			if echo "${tag}" | grep -oP '.*-rc0$'; then
+				info "This is a rc0 for master - creating stable branch"
+				stable_branch=$(echo ${tag} | awk 'BEGIN{FS=OFS="."}{print $1 "." $2}')
+				stable_branch="stable-${stable_branch}"
+				info "crating branch ${stable_branch} for ${repo}"
```
Comment (GabyCT) [inline release/tag_repos.sh]: creating?

Comment (egernst) [reply release/tag_repos.sh]: +1

**Pull request merged**